### ISSUE TITLE
How to use one GPU for training? How to adjust the parameters?

### ISSUE THREAD

**Issue by tinaZZer** [microsoft/transductive-vos.pytorch#7]
How to use one GPU for training? How to adjust the parameters?

**Comment by eastOffice** [comment]
Hello,

We haven't trained the model on a single GPU. You could decrease the batchsize and try to linear scale learning rate.
Specifically, you could try to set the batch size and learning rate four times smaller, i.e, batch size to 4, and learning rate to 0.005.
